src/ResourceManagement/Resource/Microsoft.Azure.Management.ResourceManager/Generated/Models/DeploymentProperties.cs

```diff
@@ -29,7 +29,7 @@ public partial class DeploymentProperties
         /// <summary>
         /// Initializes a new instance of the DeploymentProperties class.
         /// </summary>
-        public DeploymentProperties(object template = default(object), TemplateLink templateLink = default(TemplateLink), object parameters = default(object), ParametersLink parametersLink = default(ParametersLink), DeploymentMode? mode = default(DeploymentMode?), DebugSetting debugSetting = default(DebugSetting))
+        public DeploymentProperties(DeploymentMode? mode, object template = default(object), TemplateLink templateLink = default(TemplateLink), object parameters = default(object), ParametersLink parametersLink = default(ParametersLink), DebugSetting debugSetting = default(DebugSetting))
         {
```

Comment:
+1 on Gaurav's comment. Is this generated code? can we set the default value to "default(DeploymentMode)" which is "Incremental"?